src/libPMacc/include/particles/AsyncCommunicationImpl.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2015 Alexander Grund
```

Comment:
This list should contain all authors of src/libPMacc/include/particles/ParticlesBase.tpp because most of the file is copied.